src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Main program for Duke.
+ */
+public class Duke {
+    /**
+     * Main program function.
```

Comment:
Minor coding style issue, think that there should be a line break between the description and @param (according to the code snippet in the java coding standard). Noticed these java doc formatting issue elsewhere as well.